evm_arithmetization/Cargo.toml

```diff
@@ -47,6 +48,7 @@ serde_json = { workspace = true }
 # Local dependencies
 mpt_trie = { workspace = true }
 zk_evm_proc_macro = { workspace = true }
+alloy.workspace = true
```

Comment:
We cannot include `alloy` in `evm_arithmetization` or we won't be able to release new packages. Instead we need to keep `ethereum-types` until #229 is addressed, and rely on the compatibility module in higher levels.

Reply:
Using alternative now, thanks.